### DISCUSSION TITLE
Should the `organizeImports` rule be highlighted in VSCode?

### DISCUSSION THREAD

**Discussion by fax1ty** [biomejs/biome#263] [Polls]
I think it would be very convenient to have a way to automatically fix the imports.
Also, it is very useful when working in a team to immediately visually spot a mistake in accordance with the established rules.

How it works in ESLint (using a plugin):

![image](https://github.com/biomejs/biome/assets/26363214/1d365d0b-b215-4d5d-bc72-762bee2159a5)

**Comment by Conaclos** [comment]
I added the option `I don't mind`. In the process, I certainly removed your vote. I am sorry.

**Comment by willruggiano** [comment]
Make it a configuration option and everyone will be happy :)
